aws/kinesis/core/shard_map.cc

```diff
@@ -97,7 +97,10 @@ void ShardMap::update() {
 
 void ShardMap::list_shards(const Aws::String& next_token) {
   Aws::Kinesis::Model::ListShardsRequest req;
+  Aws::Kinesis::Model::ShardFilter filter;
+  filter.SetType(Aws::Kinesis::Model::ShardFilterType::AT_LATEST);
   req.SetMaxResults(1000);
+  req.SetShardFilter(filter);
```

Comment:
I don't think `ShardFilter` is settable on a paginated request (I believe this will break with a 400), can you confirm?

Reply:
sounds good will do in next revision